### ISSUE TITLE
[A11y] Insights No Keyword Search Results Grid View

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#3298]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Grid View
Link: https://oregondigital.org/catalog?locale=en&q%5B%5D=&search_field=all_fields&utf8=%E2%9C%93&view=gallery
No Keyword

I used Insights to validate the page, recieved 20 errors regarding the following see example:

<img width="998" alt="Image" src="https://github.com/user-attachments/assets/dd07868a-3cb2-42c6-9abd-9a25ea6f6149" />

### Suggested Fix

Use Chrome Insights to validate the page and see result errors for suggested fix.

Example for above:

Fix the following:
Element is in tab order and does not have accessible text
Fix ONE of the following:
Element does not have text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Use Chrome Insights to revalidate the page for errors.
